stock_delivery_notification/data/mail_template_data.xml

```diff
@@ -4,6 +4,7 @@
     <record id="email_template_edi_delivery" model="mail.template">
         <field name="name">Delivery Order - Send by Email</field>
         <field name="email_from">sales@xotic.us</field>
```

Comment:
Should this be the sender?  Try using object reference in any case instead of directly putting in the address.